### ISSUE TITLE
Gtk-Message: Failed to load module "atk-bridge" when running in GUI mode

### ISSUE THREAD

**Issue by bitsgalore** [EasyinnovaSL/DPFManager#378]
When I run DPFmanager in GUI mode, it always gives me the following message: 

   Gtk-Message: Failed to load module "atk-bridge"

Other than that, the GUI appears to work normally. 

Platform: Linux Mint 18.2; installed from <http://www.blueroominnovation.com/dpfmanager/Downloads/Current-release/Ubuntu/dpf-manager-3.1.1.deb>

**Comment by victormunoz** [comment]
Yes, this message appears in some Linux systems, we are still not sure why.
But from our testing, it seems to be harmless.
